ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -120,6 +120,48 @@ Item {
                        + bannerColumn.height
                        + Style.current.bigPadding
 
+        background: MouseArea {
+            acceptedButtons: Qt.RightButton
+            onClicked: {
+                popup.x = mouse.x
+                popup.y = mouse.y
+                popup.open()
```

Comment:
We're opening context menus 4px below the cursor. Can you update this accordingly?